.changeset/yellow-frogs-help.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/ui-extensions': minor
+'@shopify/ui-extensions-react': minor
+---
+
+Add checkoutToken api
```

Comment:
Can we make this more descriptive? This will ultimately be part of a public changelog and we want to start putting more effort into these. For example, what problem does this resolve/feature does it unlock?

Reply:
Updated.